src/ApiPort/ApiPort.Vsix/ApiPort.Vsix.csproj

```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ApiPort.Vsix</RootNamespace>
     <AssemblyName>ApiPort.Vsix</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
*This* TFM is meaningless. Notice the `IncludeAssemblyInVSIXContainer=false` property below. So you're not limited the VS's that this can run on at all by changing this.
But by changing this, you stop getting build warnings when this project references the VS2019 project which *does* need to reference net472.